vehicle/mkvi/software/sensing/sensing.c

```diff
@@ -0,0 +1,69 @@
+/*
+Header:
+    Code for the Sensing Controller Board located in the nostril motherboard and
+the rear sensing box Author:
+    @author Richard Li
+    @date   03/25/2023
+*/
+
+/*----- Includes -----*/
+#include "sensing.h"
+#include "can_api.h"
+#include "log_uart.h"
+#include "utils/timer.h"
+#include <avr/interrupt.h>
+#include <avr/io.h>
+#include <stdio.h>
+#include <string.h>
+#include <util/delay.h>
+
+void timer1_isr(void) {
+    send_can = true;
+};
+
+int pcint0_callback(int counter_left) {
+    // Left wheel speed sensor is in pin PB2 -> PORTB
+    sensing.wsl = !gpio_get_pin(WHEEL_SPEED_LEFT);
+    if (sensing.wsl) {
+        counter_left += 1;
+    }
+}
+void pcint2_callback(int counter_right) {
+    // Right wheel speed sensor is in pin PB2 -> PORTB
+    sensing.wsr = !gpio_get_pin(WHEEL_SPEED_RIGHT);
+    if (sensing.wsr) {
+        counter_right += 1;
+    }
+}
+
+int main(void) {
+    can_init_sensing();
+    adc_init();
+    CAN_init(CAN_ENABLED);
```

Comment:
What is this line doing? Don't think we need it for CAN API

Reply:
No ADCs for now.

Reply:
I was referring to `CAN_init(CAN_ENABLED);`, but yeah nice catch on the ADCs